clients/drcachesim/tests/memref_gen.h

```diff
@@ -137,6 +146,34 @@ gen_exit(memref_tid_t tid)
     return memref;
 }
 
+inline std::vector<memref_t>
```

Comment:
From offline discussion: document the input requirements: have to set tid + pid fields but not the others; have to fill in all operand records

Reply:
Done.